src/pages/Login/Forgotpassword.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Forgotpassword.scss';
+
+class Forgotpassword extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+    };
+  }
+
+  handleInputValueChange = e => {
+    const { id, value } = e.target;
+    this.setState({
+      [id]: value,
+    });
+  };
+
+  checkValidation = e => {
+    e.preventDefault();
+    const { id } = this.state;
+    if (id) {
+      alert('success');
+    } else {
+      alert('Fail');
+    }
```

Comment:
```suggestion
    id ? alert('success') : alert('Fail');
```
삼항연산자로 처리 가능할 것 같습니다.

Reply:
넵